mh/pat_item.py

```diff
@@ -131,6 +131,13 @@ def unpack_optional_fields(data, offset=0):
     return info
 
 
+def pack_unk_short_array_struct(capcom_id, name):
```

Comment:
I don't think this should be made into a function, since this is only creating the payload for the `NtcLayerHost` packet.

It should be okay to create a function that assemble and send the packet. Something like 
```python
def sendNtcLayerHost(new_leader, seq):
    """NtcLayerHost packet.
    ID: 64411000
    JP: レイヤのホスト通知
    TR: Layer host notification
    """

    data = struct.pack(">HII", 0x08, 0x00, 0x00)  # unk short array pat item
    data += pati.lp2_string(new_leader.capcom_id)
    data += pati.lp2_string(new_leader.name)
    
    self.server.layer_broadcast(self.session, PatID4.NtcLayerHost, data, seq)

```
Also this function should be on `fmp_server.py`

Reply:
This has been fixed in a recent commit.